R/model.R

```diff
@@ -520,23 +520,29 @@ CmdStanModel$set("public", name = "compile", value = compile_method)
 #' @section Usage:
 #'   ```
 #'   $check_syntax(
-#'     quiet = FALSE,
+#'     pedantic = FALSE,
 #'     include_paths = NULL,
-#'     stanc_options = list()
+#'     stanc_options = list(),
+#'     quiet = FALSE
```

Comment:
I rearranged these arguments a bit. I think for check_syntax `quiet` is probably only used by packages like brms so it doesn't need to be the first argument.

Reply:
Good call.